content/data-types/studio/_index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Studio: Area of a Circle"
+date: 2023-06-14T09:27:57-05:00
+draft: false
+weight: 3
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: Courtney Frey # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Get cozy with Java syntax by writing a console program that calculates the area of a circle based on input from the user.
+
+## Creating your class
+Since you’re still new to Java and IntelliJ, we’ll provide some extra direction the first studio.
+
+First, make a new folder, or package, to hold your studio work. Create a new package named `org.launchcode.java.studios.areaofacircle` by right-clicking (or ctrl-clicking for some Mac users) on the `src` directory in `java-web-dev-exercises` and selecting _New_ > _Package_. Be sure to enter the full name, or your package won’t be created in the correct location.
+
+Create your class in the `java-web-dev-exercises` project within the package `org.launchcode.java.studios.areaofacircle` by right-clicking/ctrl-clicking on the `studios.areaofacircle` package/folder and selecting _New_ > _Java Class_. Enter the name Area. Select the option to add the file to Git when the window appears.
```

Comment:
Suggestion to simplify by making instructions an ordered list like in exercises?  Please update based on the new repo name.

Reply:
check!